salt/addons/dns/init.sls

```diff
@@ -1,4 +1,4 @@
-{% if salt.caasp_pillar.get('addons:dns', False) %}
+{% if salt.caasp_pillar.get('addons:dns', True) %}
```

Comment:
This was defaulting in the static pillar to True - sync'd it up while I noticed it.